<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberSetting</class>
 <widget class="QDialog" name="MemberSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>-10</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kenyan Coffee</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Member's Setting</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>391</width>
     <height>375</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="idLabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="usernameLabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username: </string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="passwordButton">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="passwordEdit">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="eventAmtLabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string># of events: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="nameLabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password: </string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="hourLabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total Hours: </string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="gradeLabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Grade: </string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="awardGained">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CSA Award:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
